src/openforms/formio/tests/validation/test_time.py

```diff
@@ -0,0 +1,73 @@
+from django.test import SimpleTestCase
+
+from openforms.typing import JSONValue
+
+from ...typing import Component
+from .helpers import extract_error, validate_formio_data
+
+
+class TimeFieldValidationTests(SimpleTestCase):
+
+    def test_timefield_required_validation(self):
+        component: Component = {
+            "type": "time",
+            "key": "foo",
+            "label": "Foo",
+            "validate": {"required": True},
+        }
+
+        invalid_values = [
+            ({}, "required"),
+            ({"foo": None}, "null"),
+        ]
+
+        for data, error_code in invalid_values:
+            with self.subTest(data=data):
+                is_valid, errors = validate_formio_data(component, data)
+
+                self.assertFalse(is_valid)
+                self.assertIn(component["key"], errors)
+                error = extract_error(errors, component["key"])
+                self.assertEqual(error.code, error_code)
+
+    def test_min_max_time(self):
+        component: Component = {
+            "type": "time",
+            "key": "foo",
+            "label": "Foo",
+            "validate": {
+                "required": False,
+                "minTime": "10:00",
```

Comment:
probably to have a case with seconds specified too, for legacy components (the new builder outputs hours and minutes only, but the old one did more I think)